dev/com.ibm.ws.microprofile.faulttolerance_fat_tck/bnd.bnd

```diff
@@ -19,6 +19,9 @@ src: \
 
 fat.project: true
 
+tested.features:\
```

Comment:
if this sometimes runs on ee7 then doesnt the tested.features need to include cdi-1.2 and servlet-3.1?

Reply:
I believe the way it works is this is for features that are not in any of the server.xml files but are swapped in later. At the moment it's set up to start on EE7 and move to EE8 so cdi-1.2 and servlet-3.1 are in the server.xml.